src/joint_state_reader.cpp

```diff
@@ -12,7 +12,7 @@
 namespace rapid {
 namespace pbd {
 JointStateReader::JointStateReader()
-    : nh_(), topic_("/joint_states"), joint_state_() {}
+    : nh_(), topic_("/robot/joint_states"), joint_state_() {}
```

Comment:
Don't use the Baxter topic as the default. I think Baxter is probably the outlier.